### PR TITLE
release artifacts for release v0.0.2

### PR DESCRIPTION
**Pull request by jaypipes**
Updated release artifacts for v0.0.2 release.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: services/apigatewayv2/helm/templates/role-writer.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+kind: Role
+metadata:
+  creationTimestamp: null
+  name: ack-apigatewayv2-writer
+  namespace: {{ .Release.Namespace }}
+rules:
+- apiGroups:
+  - apigatewayv2.services.k8s.aws
+  resources:
+  - apis
```
Comment (surajkota) [inline services/apigatewayv2/helm/templates/role-writer.yaml:12]: nit: extra lines here and in a few other templates, is this expected?

**Review by A-Hilaly — APPROVED** [review]
/lgtm 👍

**Review by vijtrip2 — APPROVED** [review]

**Review by surajkota — APPROVED** [review]

**Comment by kumargauravsharma** [comment]
It seems the `service-acount` related yamls have not been generated for services.

The template for it:
https://github.com/aws/aws-controllers-k8s/blob/main/templates/helm/templates/service-account.yaml#L12-L16
contains `annotations` 

But currently the latest service-account.yaml under service helm directory does not contain it; and needs to be auto generated.
https://github.com/aws/aws-controllers-k8s/blob/main/services/elasticache/helm/templates/service-account.yaml#L12-L13

Would service-account.yaml files for services be generated in next steps of release scripts?

**Review by kumargauravsharma — CHANGES_REQUESTED** [review]
Please refer my comment on `service-acount` and `annotations`

File: services/apigatewayv2/helm/templates/role-writer.yaml (hunk 1 above)
Comment (jaypipes) [reply services/apigatewayv2/helm/templates/role-writer.yaml:12]: nice catch. I can fix this up.

Comment (jaypipes) [reply services/apigatewayv2/helm/templates/role-writer.yaml:12]: I will fix this in a future PR.

**Review by kumargauravsharma — APPROVED** [review]

**Pull request merged**